scripts/generate_schema/worldbank/generate_worldbank_schema.py

```diff
@@ -0,0 +1,126 @@
+        data_ind = resdata.json()
+        # print(data_ind)
+        materialiseFormat = 'csv'
+        infoFormat = 'json'
+        print("Generating schema for Trading economics", commondata['name'])
+        schema = {}
+        schema["title"] = commondata['name']
+        schema["description"] = commondata['sourceNote']
+        schema["url"] = "https://api.worldbank.org/v2/indicators/" + commondata['id'] + "?format=json"
+        schema["keywords"] = [i for i in commondata['name'].split()]
+        schema["date_updated"] = data_ind[0]["lastupdated"]
+        schema["license"] = None
+        schema["provenance"] = "http://worldbank.org"
+        schema["original_identifier"] = commondata['id']
+
+        schema["materialization"] = {
+            "python_path": "worldbank_materializer",
+            "arguments": {
+                "url": "https://api.worldbank.org/v2/indicators/" + commondata['id'] + "?format=json"
+            }
+        }
+        schema['variables'] = []
+        first_col = {
+            "name": "indicator.id",
+
+            "description": "id is identifier of an indicator in worldbank datasets",
+            "semantic_type": ["https://metadata.datadrivendiscovery.org/types/CategoricalData"]
+        }
+        second_col = {
+            "name": "indicator.value",
+
+            "description": "name of an indicator in worldbank datasets",
+            "semantic_type": ["http://schema.org/Text"]
+        }
+        third_col = {
+            "name": "unit",
+
+            "description": "unit of value returned by this indicator for a particular country",
+            "semantic_type": ["https://metadata.datadrivendiscovery.org/types/CategoricalData"]
+
+        }
+        fourth_col = {
+            "name": "sourceNote",
+
+            "description": "Long description of the indicator",
+            "semantic_type": ["http://schema.org/Text"]
+        }
+        fifth_col = {
+            "name": "sourceOrganization",
+            "description": "Source organization from where Worldbank acquired this data",
+            "semantic_type": ["http://schema.org/Text"]
+        }
+        sixth_col = {
+            "name": "country.value",
+
+            "description": "Country for which idicator value is returned",
+            "semantic_type": ["https://metadata.datadrivendiscovery.org/types/Location"],
+            "named_entity": None
+        }
+        seventh_col = {
+            "name": "countryiso3code",
+
+            "description": "Country iso code for which idicator value is returned",
+            "semantic_type": ["https://metadata.datadrivendiscovery.org/types/Location"]
+        }
+        eighth_col = {
+            "name": "date",
+
+            "description": "date for which indictor value is returned for a particular country",
+            "semantic_type": ["https://metadata.datadrivendiscovery.org/types/Time"],
+            "temporal_coverage": {"start": None, "end": None}
```

Comment:
@BhavyaGulati For this column. Is it possible for you to know what is the time range of each dataset covers?

Reply:
Yes, that is doable. For now I was just fetching single page of data for values (as we just needed the structure of columns). But if we need a time range, then I will have to collect all of the data.